skglm/tests/test_group.py

```diff
@@ -106,6 +107,36 @@ def test_vs_celer_grouplasso(n_groups, n_features, shuffle):
     grp_indices, grp_ptr, groups = _generate_random_grp(n_groups, n_features, shuffle)
     weights = abs(rnd.randn(n_groups))
 
+    alpha_max = _alpha_max_group_lasso(X, y, n_groups, grp_indices, grp_ptr, weights)
```

Comment:
@PABannier n_groups can be recovered from `grp_ptr.shape[0] - 0` so we can get rid of this parameter to make the function simpler